src/main/java/com/example/doodle_clone/models/Meeting.java

```diff
@@ -22,18 +31,17 @@ public class Meeting {
 
     private List<Slot> slots;
 
-    private String title, text, location;
-    private int count, IDadmin, idUsers;
+    @NonNull private String title;
```

Comment:
This is strange, because some sources write that variables must be `final` for @RequiredArgsConstructor, some - @NotNull,not null worked